content/news/Pan-STARRS-data_HLSP.md

```diff
@@ -0,0 +1,12 @@
+Title: Release of Pan-STARRS C16 and C17 sparse time series photometry
```

Comment:
Let's remove the word `sparse`, because it's probably one of the most "un-sparse" ground follow-up of a K2 field so far!